plugins/modules/dcnm_bootflash.py

```diff
@@ -361,29 +350,69 @@ def raise_type_error(msg):
             raise TypeError(f"{self.class_name}.{method_name}: {msg}")
 
         for switch in self.switches:
-            if switch.get("ip_address", None) is None:
+            if not switch.get("ip_address"):
                 msg = "Expected ip_address in switch dict. "
                 msg += f"Got {switch}."
                 raise_value_error(msg)
 
-            if switch.get("targets", None) is None:
+            if not switch.get("targets", []):
                 switch["targets"] = self.targets
             if not isinstance(switch["targets"], list):
                 msg = "Expected list of dictionaries for switch['targets']. "
                 msg += f"Got {type(switch['targets']).__name__}."
                 raise_type_error(msg)
 
-            for target in switch["targets"]:
-                if target.get("filepath", None) is None:
+            target: dict[str, str]
+            for target in switch.get("targets", []):
+                if not target.get("filepath"):
                     msg = "Expected filepath in target dict. "
                     msg += f"Got {target}."
                     raise_value_error(msg)
-                if target.get("supervisor", None) is None:
+                if not target.get("supervisor"):
                     msg = "Expected supervisor in target dict. "
                     msg += f"Got {target}."
                     raise_value_error(msg)
             self.want.append(copy.deepcopy(switch))
 
+    @property
+    def rest_send(self) -> RestSend:
+        """
+        ### Summary
+        An instance of the RestSend class.
+
+        ### Raises
+        -   getter: `ValueError` if rest_send has not been properly initialized (missing params).
+        -   setter: ``TypeError`` if the value is not an instance of RestSend.
+
+        ### getter
+        Return a properly initialized instance of the RestSend class.
+
+        ### setter
+        Set an instance of the RestSend class.
+        """
+        if not self._rest_send.params:
+            msg = f"{self.class_name}.rest_send: "
+            msg += "rest_send has not been set (missing params)."
+            raise ValueError(msg)
+        return self._rest_send
+
+    @rest_send.setter
+    def rest_send(self, value: RestSend):
+        method_name = inspect.stack()[0][3]
```

Comment:
I see other places where you are using type hints for `method_name`.  Is there a reason you are not doing it here?